lib/internal/cluster/worker.js

```diff
@@ -3,8 +3,7 @@ const EventEmitter = require('events');
 const internalUtil = require('internal/util');
 const util = require('util');
 const defineProperty = Object.defineProperty;
-const suicideDeprecationMessage =
-    'worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.';
+const deprecationMessage = 'Please use worker.exitedAfterDisconnect instead.';
```

Comment:
This will cause people to see warnings like `(node:4680) DeprecationWarning: Please use worker.exitedAfterDisconnect instead.`, which is extremely confusing without context.